### DISCUSSION TITLE
configureable exports like in invoice section

### DISCUSSION THREAD

**Discussion by Martin-Forster** [kimai/kimai#2713] [General]
Dear Sirs,

to communicate the data into the billing system we would like to have configureable exports like in the invoice section.
Invoice is usable but troublesome for our usecase.
Because:
- we need the ordernumber from the project definiton
- we need to create a invoice per project, because of that.
- we can't automate it

We would like to have a export per month with all billable entries, and the according project order number.
with
- customizeable columnnames
- values from the customfields plugin
- accessible by api, for automation

Summary:
We would like to have the redering options from invoice together with the more flexible data access from export.

Thanks for considering.

**Comment by kevinpapst** [comment]
HI Martin,

as you want to use the API anyways, why don't you use the Timesheet API with the `full=true` flag to export all timesheet records?
